acbuild/acbuild.go

```diff
@@ -239,6 +244,12 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 }
 
 func main() {
+	if path.Base(os.Args[0]) == "acbuild-script" {
+		os.Args[0] = path.Join(path.Dir(os.Args[0]), "acbuild")
+		args := os.Args[1:]
+		os.Args = []string{os.Args[0], "script"}
+		os.Args = append(os.Args, args...)
+	}
```

Comment:
(Sorry for having been cryptic) Yes! It probably won't solve the issue here (argv[0] adjusting for later re-exec) but I think it will help in better integrating with multicall machinery.